client/web/app/elements/rest-services/lectures-service.html

```diff
@@ -0,0 +1,69 @@
+<dom-module id="lectures-service">
+    <template>
+        <iron-ajax
+                id="ajax"
+                last-response="{{lastResponse}}"
+        ></iron-ajax>
+    </template>
+    <script>
+        Polymer({
+            is: "lectures-service",
+            properties:{
+                lastResponse: {
+                    type: Object,
+                    notify: true,
+                    readOnly: false
+                }
+            },
+            queryAll: "/lectures",
+            query: function(search) {
+                var url = Config.urls.backend + this.queryAll;
+                if (StringUtils.isNotEmpty(search)) {
+                    url += "?search=" + search;
+                }
+                this.$.ajax.url = url;
+                this.$.ajax.method = "GET";
+                this.$.ajax.generateRequest();
+            },
+            queryByCategory: function(category, limit) {
+                var url = Config.urls.backend + this.queryAll + "?category=" + category + "&limit=" + limit;
+                this.$.ajax.url = url;
+                this.$.ajax.method = "GET";
+                this.$.ajax.generateRequest();
+            },
+            queryByAuthor: function(author, limit) {
+                var url = Config.urls.backend + this.queryAll + "?author=" + author + "&limit=" + limit;
+                this.$.ajax.url = url;
+                this.$.ajax.method = "GET";
+                this.$.ajax.generateRequest();
+            },
+            create: function(data){
+                console.log("data for post: ", data);
+                this.$.ajax.url = Config.urls.backend + this.queryAll;
+                this.$.ajax.body = data;
+                this.$.ajax.contentType = "application/json";
+                this.$.ajax.method = "POST";
+                this.$.ajax.generateRequest();
+            },
+            delete: function(lectureId){
+                console.log("data for delete: ", lectureId);
+                this.$.ajax.url = Config.urls.backend + this.queryAll + "/" + lectureId;
+                this.$.ajax.method = "DELETE";
+                this.$.ajax.generateRequest();
+            },
+            get: function(lectureId){
+                this.$.ajax.url = Config.urls.backend + this.queryAll + "/" + lectureId;
+                this.$.ajax.method = "GET";
+                this.$.ajax.generateRequest();
+            },
+            update: function(lectureId, data) {
+                console.log("Update " + lectureId, data);
```

Comment:
przyznam że nie lubię logowania komunikacji bo strzasznie to zaśmieca logi a w sumie nie jest za często przydatne. Jak chcesz to zostaw ale wydaje mi się też że tak się po prostu nie powinno robić bo to co się wysyła zawsze można zobaczyć w consoli w zakładce Network